### ISSUE TITLE
Working with Angular 9?

### ISSUE THREAD

**Issue by chrisdmacrae** [microsoft/angular-react#155]
Does `@angular-react/fabric` work with Angular 9?

Playing around, and it appears children aren't working with the new version. I get an empty react-content element where I'd expect child elements to display.

**Comment by fergusg** [comment]
I got it working by disabling Ivy.   Not good going forward as I would expect that option to be removed in a version or two.

**Comment by chrisdmacrae** [comment]
No Ivy is definitely a challenge, as it really slips down the bundles.

**Comment by MJomaa** [comment]
No it is not working with Ivy.

**Comment by gotham64** [comment]
Please we need this

**Comment by gineign** [comment]
also facing this issue

**Comment by eran-pinhas** [comment]
as @fergusg said - disableing ivy with `"angularCompilerOptions": {"enableIvy": false }` in tsconfig.json seems to work

**Comment by gotham64** [comment]
But then it give up the reason for using angular :/ @eran-pinhas

**Comment by MJomaa** [comment]
@gotham64 True, we switched to React because Fluent UI just works better with it.

**Comment by gineign** [comment]
Will you be working with this issue?

**Comment by Eve-Sama** [comment]
Nowadays, Angular's latest version is V12. Anyone use this library in V12?
